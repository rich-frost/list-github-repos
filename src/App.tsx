import Header from './components/Header';
import Main from './components/Main';

// TODO add integration tests to cover e2e logic
export default function App() {
  return (
    <>
      <Header />
      <Main />
    </>
  );
}
