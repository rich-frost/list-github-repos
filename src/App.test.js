import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

it('renders the app', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
