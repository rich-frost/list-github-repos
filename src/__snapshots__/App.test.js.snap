// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the app 1`] = `
<h1>
  Hello world
</h1>
`;
